client/services.go

```diff
@@ -598,3 +599,12 @@ type WorkspacesClient interface {
 	DescribeWorkspaces(ctx context.Context, params *workspaces.DescribeWorkspacesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspacesOutput, error)
 	DescribeWorkspaceDirectories(ctx context.Context, params *workspaces.DescribeWorkspaceDirectoriesInput, optFns ...func(*workspaces.Options)) (*workspaces.DescribeWorkspaceDirectoriesOutput, error)
 }
+
+//go:generate mockgen -package=mocks -destination=./mocks/backup.go . BackupClient
+type BackupClient interface {
```

Comment:
Missing ListTags API https://docs.aws.amazon.com/aws-backup/latest/devguide/API_ListTags.html this for vault/plan/recovery point resoruces.

Reply:
Added tags.